app.js

```diff
@@ -1,17 +1,67 @@
 var express = require('express');
 var databaseConfig = require('./config/databaseConfig');
+var flash = require('connect-flash');
+var passport = require('passport'),
+  LocalStrategy = require('passport-local').Strategy;
+var session = require('express-session');
 var app = express();
-
 var mongoose = require('mongoose');
 mongoose.connect(databaseConfig.getMongoURI());
 
 app.use(express.static('public'));
 app.use(express.static('node_modules'));
+app.use(flash());
+app.use(session({
+  secret: 'secretkey1'
+}));
+app.use(passport.initialize());
+app.use(passport.session());
+passport.use(new LocalStrategy(
+  function(username, password, done) {
+    User.findOne({
```

Comment:
This whole function is boilerplate copied from passportjs docs as a basic example. The `User.findOne` function call will be related to a Mongoose kind of object (User) that has a function call that accesses the mongo database and finds the user with the given `username`. The `done` functions are part of passportjs.